thirdParty/mdspan/examples/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.12...3.19)
+
+project(example LANGUAGES CXX)
+
+if(NOT TARGET std::mdspan)
+  find_package(mdspan REQUIRED)
+endif()
+
+function(mdspan_add_example EXENAME)
+  add_executable(${EXENAME} ${EXENAME}.cpp)
+  target_link_libraries(${EXENAME} std::mdspan)
```

Comment:
`find_package` and `target_link_libraries` for `mdspan` should not be required because alapka should handle it.

Reply:
These are examples from the mdspan repository, not my code :)